### PR TITLE
add help command for proxy start up script

### LINKED ISSUE #6273: Add shardingsphere-proxy start scripts help command
## Feature Request

The shardingsphere-proxy  doesn't have any more information about how to start, what is the optional parameter， 
I want to add start scripts help function to show more convenience, 

examples like: 

bin/start.sh -h  or  bin/start.sh --help

### PR DESCRIPTION
**Pull request by xbkaishui**
Fixes #6273 

Changes proposed in this pull request:
- add help command

examples: 

sh bin/start.sh -h

usage: start.sh [port] [config_dir]
  port: proxy listen port, default is 3307
  config_dir: proxy config directory, default is conf

### CODE REVIEW

File: shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat [hunk 1]
```diff
@@ -27,6 +27,13 @@ set PORT=%1
 
 set CONFIG=%2
 
+if "%PORT%"=="-h" (
+    goto print_usage
+)
+if "%PORT%"=="-help" (
```
Comment (tuohai666) [inline shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat:33]: original intention --help?

File: shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat [hunk 2]
```diff
@@ -27,6 +27,13 @@ set PORT=%1
 
 set CONFIG=%2
 
+if "%PORT%"=="-h" (
```
Comment (tuohai666) [inline shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat:30]: Why .bat is different from .sh and use goto?

File: shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.sh [hunk 3]
```diff
@@ -38,6 +38,17 @@ JAVA_MEM_OPTS=" -server -Xmx2g -Xms2g -Xmn1g -Xss256k -XX:+DisableExplicitGC -XX
 
 MAIN_CLASS=org.apache.shardingsphere.proxy.Bootstrap
 
+print_usage() {
```
Comment (tuohai666) [inline shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.sh:41]: there's another start.sh file. Please find it.

File: shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat (hunk 1 above)
Comment (tuohai666) [inline shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat:33]: Why .bat is different from .sh and use goto?

Comment (xbkaishui) [reply shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat:33]: sorry miss -

File: shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat (hunk 2 above)
Comment (xbkaishui) [reply shardingsphere-distribution/shardingsphere-proxy-distribution/src/main/resources/bin/start.bat:30]: windows bat is a little difference with bash,  it use goto as a function call

**Pull request merged**